patches.go

```diff
@@ -102,10 +102,14 @@ func toMergeOption(p PatchInterface) (mo *xpv1.MergeOptions, err error) {
 	switch pp.GetToFieldPathPolicy() {
 	case v1beta1.ToFieldPathPolicyReplace:
 		// nothing to do, this is the default
-	case v1beta1.ToFieldPathPolicyAppendArray:
+	case v1beta1.ToFieldPathPolicyAppendArray, v1beta1.ToFieldPathPolicyForceMergeObjectsAppendArrays:
 		mo = &xpv1.MergeOptions{AppendSlice: ptr.To(true)}
-	case v1beta1.ToFieldPathPolicyMergeObject:
+	case v1beta1.ToFieldPathPolicyMergeObject, v1beta1.ToFieldPathPolicyMergeObjects:
 		mo = &xpv1.MergeOptions{KeepMapValues: ptr.To(true)}
+	case v1beta1.ToFieldPathPolicyForceMergeObjects:
+		mo = &xpv1.MergeOptions{KeepMapValues: ptr.To(false)}
+	case v1beta1.ToFieldPathPolicyMergeObjectsAppendArrays:
+		mo = &xpv1.MergeOptions{KeepMapValues: ptr.To(true), AppendSlice: ptr.To(true)}
```

Comment:
I'd find this a little easier to parse if it was sorted:

1. Replace
2. MergeObjects, MergeObject
3. MergeObjectsAppendArrays
4. ForceMergeObjects
5. ForceMergeObjectsAppendArrays, AppendArray

Reply:
Thanks, addressed.